apps/frontend/src/components/ui/servers/marketing/ServerPlanSelector.vue

```diff
@@ -110,78 +135,93 @@ const sharedCpus = computed(() => {
-          <p class="m-0">{{ sharedCpus }} Shared CPUs</p>
-        </div>
-        <div class="flex items-center gap-2 text-secondary">
-          <SparklesIcon /> Bursts up to {{ cpus }} CPUs
-          <nuxt-link
-            v-tooltip="
-              `CPU bursting allows your server to temporarily use additional threads to help mitigate TPS spikes. Click for more info.`
-            "
-            to="/servers#cpu-burst"
-            @click="() => emit('scroll-to-faq')"
-          >
-            <UnknownIcon class="h-4 w-4 text-secondary opacity-80" />
-          </nuxt-link>
-        </div>
         <span class="m-0 text-2xl font-bold text-contrast">
-          ${{ price / 100 }}<span class="text-lg font-semibold text-secondary">/month</span>
+          ${{ price > 0 ? (price / 100).toFixed(2) : '0.00' }}<span class="text-lg font-semibold text-secondary">
+            /month
+          </span>
         </span>
+        <p class="m-0 text-base text-secondary leading-normal max-w-xs">{{ formatMessage(plans[plan].description) }}</p>
       </div>
+
       <ButtonStyled
         :color="plans[plan].buttonColor"
         :type="plan === 'medium' ? 'standard' : 'highlight-colored-text'"
         size="large"
+        class="w-full mt-auto mb-4" 
       >
-        <span v-if="outOfStock" class="button-like disabled"> Out of Stock </span>
-        <button v-else @click="() => emit('select')">
-          Get Started
-          <RightArrowIcon class="shrink-0" />
+        <span v-if="outOfStock" class="button-like disabled w-full"> Out of Stock </span>
+        <button v-else class="w-full" @click="() => emit('select')">
+          Select Plan
         </button>
       </ButtonStyled>
+
+      <ul class="m-0 flex flex-col gap-2 p-0 text-sm text-secondary">
+        <li class="flex items-center gap-2">
+          <SparklesIcon class="size-4 text-brand" /> {{ formattedRam }} GB RAM
+        </li>
+        <li class="flex items-center gap-2">
+          <SparklesIcon class="size-4 text-brand" /> {{ formattedStorage }} GB SSD
+        </li>
+        <li class="flex items-center gap-2">
+          <SparklesIcon class="size-4 text-brand" /> {{ sharedCpus }} Shared CPUs
+        </li>
+        <li class="flex items-center gap-2">
+          <SparklesIcon class="size-4 text-brand" /> Bursts up to {{ cpus }} CPUs
+          <nuxt-link
+            v-tooltip="
+              `CPU bursting allows your server to temporarily use additional threads to help mitigate TPS spikes. Click for more info.`
+            "
+            to="/servers#cpu-burst"
+            class="flex items-center"
+            @click="() => emit('scroll-to-faq')"
+          >
+            <UnknownIcon class="size-4 text-secondary opacity-60 hover:opacity-100 transition-opacity" />
+          </nuxt-link>
+        </li>
+      </ul>
     </div>
   </li>
 </template>
 
-<style scoped lang="scss"></style>
+<style scoped lang="scss">
+.button-like.disabled {
```

Comment:
Should not be here, use tailwind or use the Button component, it already supports `disabled` prop